### PR TITLE
Update README.md with new information

### LINKED ISSUE #3149: README out of date
The README has outdated information.

Updates required:
* [gitter channel](https://gitter.im/PointCloudLibrary/pcl)
* Google group(?)
* Remove mailing list(?)
* Remove API Lab link

Any other suggestions?

### PR DESCRIPTION
**Pull request by kunaltyagi**
* Uniform badges for CI and the rest of README
* Badges for Gitter, SO and Website with correct information
* Update of information in README.md and CONTRIBUTION.md

Cheers

Closes #3149

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,8 +16,14 @@ the preferred channel for submitting [pull requests](#pull-requests) and
 [bug reports](#bugs), but please respect the following
 restrictions:
 
-* Please **do not** use the issue tracker for personal support requests (use
-  [mailing list](http://www.pcl-users.org/)).
+* Please **do not** use the issue tracker for personal support requests. Please choose one of the following alternatives instead:
+  * Q&A at [Stack Overflow](https://stackoverflow.com/questions/tagged/point-cloud-library):
+  Do remember to tag your questions with the tag `point-cloud-library`
+  * Live chat on the [Gitter chanel](http://www.pcl-users.org/)
```
Comment (taketwo) [inline CONTRIBUTING.md]: Please fix the link here. Also, chan**n**el.

File: README.md [hunk 2]
```diff
@@ -49,9 +61,20 @@ Please read [CONTRIBUTING.md](https://github.com/PointCloudLibrary/pcl/blob/mast
 
 Issues
 ------
-For general questions on how to use the PCL, please use the [pcl-users](http://www.pcl-users.org/) mailing list (do not forget to subscribe before posting).
+For general questions on how to use the PCL, please choose one of the following alternatives instead:
```
Comment (taketwo) [inline README.md]: I would switch the order. Start with "To report issues...", then "For general questions...".

Plus the same comment regarding link and spelling.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kunaltyagi) [reply CONTRIBUTING.md]: Oops. On it

File: README.md (hunk 2 above)
Comment (kunaltyagi) [reply README.md]: PTAL

**Comment by taketwo** [comment]
Thanks, looks good now. One question though: do we actually want to advertise Gitter for "live chat"? I feel it sends a message that there is like 24-hour live support or something, whereas in reality there is just an empty chat room :laughing:

**Comment by SergioRAgostinho** [comment]
We might not use it personally, but other users which prefer that way and can chat between themselves on point cloud trivia :). Hopefully they can support each other. It's just a matter of providing a place for people that like this type of interaction to assemble.

**Comment by taketwo** [comment]
Sure, it's good to have, just wanted to avoid setting expectations too high. But ok, maybe it's just me seeing an unfulfilled promise in this wording.

**Comment by SergioRAgostinho** [comment]
I see the issue now. We mention gitter as live chat for solving issues! That's probably not a good idea. My view on Gitter is more on the social aspect, not so much dedicated support. What's your view on this @kunaltyagi ?

**Comment by kunaltyagi** [comment]
For gitter, I don't think even I intended to imply a live chat by the developers to resolve your issues. I'll reword it to "live chat with community and casual discussions" and reword Q&A to explicitly have the words "support dealing with troubleshooting, installation and debugging".

I personally think of gitter as an irc chatroom with bling and of SO as a latency prone high quality source for troubleshooting.

Is there a way we can have other tags on SO such as "installation", various modules of pcl, "algorithm" (for finding the correct algorithm), etc? My reasoning is that under the current model of all support to SO, there'll be a lot of low effort questions, though there should be a few like that on gitter too. If so, we could create add that information on SO and README (in a section about asking help).

**Comment by SergioRAgostinho** [comment]
> Is there a way we can have other tags on SO such as "installation", various modules of pcl, "algorithm" (for finding the correct algorithm), etc? My reasoning is that under the current model of all support to SO, there'll be a lot of low effort questions, though there should be a few like that on gitter too. If so, we could create add that information on SO and README (in a section about asking help). 

I have nothing against more tags, but I have no idea which ones will be valuable or not at this stage. I would say we observe the effects of this transition to SO and in 2 or 3 months sample the open question reassess what needs to be changed.

**Comment by kunaltyagi** [comment]
I've updated the language. PTAL to see if it's better now

Cheers
